omnigibson/scenes/scene_base.py

```diff
@@ -472,7 +472,7 @@ def reset(self):
         # Reset the states of all objects (including robots), including (non-)kinematic states and internal variables.
         assert self._initial_state is not None
         self.load_state(self._initial_state)
-        og.sim.step()
+        og.sim.step_physics()
```

Comment:
I don't think we need to render here, plus if physics dt is much smaller then action_dt it could bring down performance